src/applications/coronavirus-vaccination-expansion/config/personal-information/personal-information.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import ssnUI from 'platform/forms-system/src/js/definitions/ssn';
+import { personalInformation } from '../schema-imports';
+
+export const schema = {
+  personalInformation,
+};
+
+export const uiSchema = {
+  personalInformation: {
+    firstName: {
+      'ui:title': 'First name',
+      'ui:errorMessages': {
+        required: 'Please enter your first name.',
+      },
+    },
+    middleName: {
+      'ui:title': 'Middle name',
+    },
+    lastName: {
+      'ui:title': 'Last name',
+      'ui:errorMessages': {
+        required: 'Please enter your last name.',
+      },
+    },
```

Comment:
We can probable reuse the `fullName` definition.

https://github.com/department-of-veterans-affairs/vets-website/blob/covid-vaccine-expansion/src/platform/forms/definitions/fullName.js